.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "22:00"
+  open-pull-requests-limit: 10
+  ignore:
+  - dependency-name: uikit
```

Comment:
@LukasHirt this file was created by the old dependabot - looks like we told it to ignore uikit.

I leave it to you to decide on this.

Reply:
We're already ignoring uikit in ODS because one update broke our dropdown and since we want to remove it we're not going to invest time into fixing that update. I guess having it ignored in there should be enough since it's dependency coming from ODS.